test/test-project/test-project.test.js

```diff
@@ -53,8 +54,8 @@ describe('Testing if html was generated with proper version of the template', ()
 
     //we make sure that index.html file doesn't contain any infromation about correlationId because this feature was added in html-template 0.17.0 while this test uses template 0.16.0
     expect(isCorelationIdInHtml).toStrictEqual(true);
-    expect(console.log).toHaveBeenCalledWith(templateInstallStartedMsg);
-    expect(console.log).toHaveBeenCalledWith(`Version of used template is ${templateVersion}.`);
+    expect(console.log).toHaveBeenCalledWith(logMessage.installationDebugMessage(logMessage.TEMPLATE_INSTALL_FLAG_MSG));
+    expect(console.log).toHaveBeenCalledWith(logMessage.tempVersion(templateVersion));
```

Comment:
`tempVersion` does not exist as far as I can see. strange that linter doesn't complain 🤔